lib/formatters/html.js

```diff
@@ -82,7 +90,8 @@ function renderMessages(messages, parentIndex) {
             severityNumber: message.severity,
             severityName: message.severity === 1 ? "Warning" : "Error",
             message: message.message,
-            ruleId: message.ruleId
+            ruleId: message.ruleId,
+            ruleUrl: ruleUrl !== null ? ruleUrl : "https://eslint.org/docs/rules"
```

Comment:
I think this default should probably be removed. The only rules hosted on eslint.org are the rules that are bundled with ESLint, which would already have the rule URL provided anyway. Previously, this formatter would report 404 errors for plugin rules, and it seems like this default only serves the purpose of retaining those 404 errors, so it's probably not necessary to maintain that level of compatibility.